src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -296,16 +300,27 @@ export class AllRecords extends React.Component<IAllRecordsProps, IAllRecordsSta
     const { filterOpened, isMapView } = this.state;
     const { siloName } = this.props;
     const elementsBesideFilter = _.slice(allRecords, 0, 4);
+    const elementsBesideFilterFirstPart = _.slice(allRecords, 0, 2);
```

Comment:
what does the first and second part stand for?

Reply:
Depending on the screen size you will display 2 or 4 record cards beside the filter, so if only 2 are displayed beside the filter the elementsBesideFilterFirstPart will be used and elementsBesideFilterSecondPart will be rendered under the filter and on large screens all 4 will be displayed beside the filter and elementsBesideFilter will be used. The elementsBesideFilter is split into 2 separate arrays and I just couldn't came up with better names for those arrays

Reply:
how about:
fourCardsBesideFilter
upperTwoCardsBesideFilter
lowerTwoCardsBesideFilter

Reply:
sure, I will rename it and push the changes